src/pages/ManageBackup/index.tsx

```diff
@@ -803,6 +804,21 @@ export default function ManageBackup(props) {
     else setShowLoader(false);
   }, [healthLoading]);
 
+//   useEffect(()=> {
+//  (async ()=> {
+//    const ownNode = {
+//       active: 0,
+//       urls: ['https://test-wrapper.bithyve.com']
+//     }
+//     const stringified =  JSON.stringify(ownNode)
+//     console.log({stringified})
+//     await AsyncStorage.setItem("OwnNodes", stringified)
+
+//   config.connectToOwnNode();
+//  })()
+    
+//   }, [])
+
```

Comment:
Would it be better to delete this?

Reply:
Yea, we'll shift this to the own node flow once it's in place.